app/views/docs/graphql.phtml

```diff
@@ -0,0 +1,529 @@
+    }
+}</code></pre></div>
+
+<p>This can be an extremely useful feature for performance, network efficiency, and app responsiveness. As the processing happens on the server, the bandwidth consumed for the request can be dramatically reduced.</p>
+
+<h3><a href="/docs/graphql#query-batching" id="query-batching">Query Batching</a></h3>
+
+<p>GraphQL allows sending multiple queries or mutations in the same request. There are two different ways to batch queries. The simplest way is to include multiple fields in a single query <b>or</b> mutation.</p>
+
+<div class="ide margin-bottom" data-lang="graphql" data-lang-label="GraphQL">
+    <pre class="line-numbers"><code class="prism language-graphql" data-prism>query GetAccountAndLocale {
+    accountGet {
+        email
+    }
+    localeGet {
+        ip
+    }
+}</code></pre></div>
+
+<p>If both field executions succeed, the response will contain a data key for each field, containing the values of the selected fields.</p>
+
+<div class="ide margin-bottom" data-lang="json" data-lang-label="JSON">
+    <pre class="line-numbers"><code class="prism language-json" data-prism>{
+    "data": {
+        "accountGet": {
+            "email": "..."
+        },
+        "localeGet": {
+            "ip": "..."
+        }
+    }
+}</code></pre></div>
+
+<p>If there was no authenticated user, the <code>accountGet</code> field would fail to resolve. In such a case the value of the data key for that field will be null, and an object will be added to the errors array instead.</p>
+
+<div class="ide margin-bottom" data-lang="json" data-lang-label="JSON">
+    <pre class="line-numbers"><code class="prism language-json" data-prism>{
+    "data": {
+        "accountGet": null,
+        "localeGet": {
+            "ip": "...",
+            "country": "..."
+        }
+    },
+    "errors": [
+        {
+            "message": "User (role: guest) missing scope (account)",
+            "path": ["accountGet"]
+        }
+    ]
+}</code></pre></div>
+
+<p>Batching with a single query or mutation has some down-sides. You can not mix and match queries and mutations within the same request unless you provide an operationName, in which case you can only execute one query per request.</p>
+
+<p>Additionally, all <b>variables</b> must be passed in the same object, which can be cumbersome and hard to maintain.</p>
+
+<p>The second way to batch is to pass an array of queries or mutations in the request. In this way, you can execute queries <b>and</b> mutations and keep variables separated for each.</p>
+
+<div class="ide margin-bottom" data-lang="json" data-lang-label="JSON">
+    <pre class="line-numbers"><code class="prism language-json" data-prism>[
+    {
+        "query": "query GetAccount { accountGet{ email } }",
+    },
+    {
+        "query": "query GetLocale { localeGet { ip } }"
+    }
+]</code></pre></div>
+
+<p>This can be extremely powerful as it allows you to execute complex actions in a single network request.</p>
```

Comment:
```suggestion
<p>This allows you to execute complex actions in a single network request.</p>
```
Again, it's documentation. I really don't want us to be so hyperbolic in our language. We should avoid giving opinions like this in docs.